src/System.IO.Compression/src/Interop/Interop.zlib.Windows.cs

```diff
@@ -52,6 +52,23 @@ internal static partial class zlib
             }
         }
 
+        internal static unsafe bool IsCrc32Available()
+        {
+            try
+            {
+                // Make a P/Invoke into zlib crc32 to ensure we're able to find and use it.
+                // If we are, then use zlib.
+                crc32(0, null, 0);
+                return true;
+            }
```

Comment:
(After reading further, it looks like the ifdef is in another file, so maybe this is fine.)